Add TOML config file option

Amir-Helali opened the pull request:
Added the feature that allows the user to provide all their optional arguments as a TOML configuration file. The user can provide the output directory option by including the key: output/o, and language by using the key: lang/l in their TOML config file. The program also exits with correct error messages and codes if the TOML file provided is not valid. Additionally, if there are optional arguments provided in the TOML file that the program does not support, they are ignored. Fixes #19

Amir-Helali committed:
Add TOML config file option

Added the feature that allows the user to provide all their optional arguments as a TOML configuration file

Namatuzio commented:
Hey @Amir-Helali would you be able to quickly add some information about how to utilize your changes in the README and -h argument? Other than that, it seems to work fine, thank you!

Namatuzio commented:
Hey @Amir-Helali I just realized I have a pretty major bug in my code, I was wondering if you could change it. In line 79 of main.py, would you be able to delete the dots before txt and md with the way I'm handling filenames, it completely breaks the program when you give the CLI a directory. Thanks a bunch!

Amir-Helali committed:
Changed based on feedback

Made the changes that were asked in the PR feedback. Updated the help message to reflect the new feature. Fixed the bug when checking for file extensions.

Namatuzio commented:
Fantastic, thank you so much!
